converter/dicom-cast/src/Microsoft.Health.DicomCast.Core/Features/Worker/FhirTransaction/Observation/ObservationConstants.cs

```diff
@@ -0,0 +1,96 @@
+// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System.Collections.ObjectModel;
+
+namespace Microsoft.Health.DicomCast.Core.Features.Worker.FhirTransaction
+{
+    /// <summary>
+    /// Dicom structured report codes used by FHIR Observation profiles
+    /// </summary>
+    public static class ObservationConstants
+    {
+        // https://www.hl7.org/fhir/terminologies-systems.html
+        public const string SctSystem = "http://snomed.info/sct";
+        public const string DcmSystem = "http://dicom.nema.org/resources/ontology/DCM";
+
+        public const string Dcm = "DCM";
+        public const string Sct = "Sct";
+
+        //------------------------------------------------------------
+        // Report codes
+        // - When you encounter these codes in a structured report, it means to create a new "Does Summary" Observation
+        //------------------------------------------------------------
+        private static readonly (string, string) RadiopharmaceuticalRadiationDoseReport = ("113500", Dcm); // (113500,DCM,"Radiopharmaceutical Radiation Dose Report")
```

Comment:
Can you strongly type this tuple with equals implemented

Reply:
Changed these constants from being tuples to instances of `DicomCodeItem`.